src/endpoints/create-release-endpoint.ts

```diff
@@ -8,7 +8,7 @@ export interface CreateReleaseEndpointInput {
   readonly title: string;
   readonly targetBranch: string;
   readonly projectTag: string;
-  readonly project: string;
+  readonly projectKeys: string[];
```

Comment:
This is a breaking change in the API but that's probably fine as Baroneza is (most likely) only used internally and only by the store team.